### PR TITLE
Add SEO component

### PR DESCRIPTION
**Pull request by Akallabet**
## Description
Improve SEO scoring by adding right metatags

## Changelog
- Add SEO component
- Add meta tags within it
- Fix Cypress Ci failure by using Chrome headless instead of Electron

## Checks

- [x] 80% test coverage

## Tested in 
- [x] Chrome
- [x] Safari

### CODE REVIEW

File: src/components/SEO/SEO.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import {Helmet, HelmetProvider} from 'react-helmet-async'
+
+const SEO = ({title}) => (
+  <HelmetProvider>
+    <Helmet>
+      <html lang="en" />
+      <title>{title}</title>
+      <meta
+        name="description"
```
Comment (ildikototh) [inline src/components/SEO/SEO.js]: I think this should be in the page template ( section or article template)  . We don't want t have same description everywhere.

File: src/components/SEO/SEO.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        content="The Palace of Westminster is one of the most iconic and significant buildings in the world, and home to one of the busiest parliaments, with more than a million visitors."
+      />
+      <meta
+        name="keywords"
+        content="Restoration and Renewal, Parliament, R&R, Parliament Restoration"
```
Comment (ildikototh) [inline src/components/SEO/SEO.js]: Should be also part of the content model as keywords array

File: src/components/SEO/SEO.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+const SEO = ({title}) => (
+  <HelmetProvider>
+    <Helmet>
+      <html lang="en" />
+      <title>{title}</title>
+      <meta
+        name="description"
+        content="The Palace of Westminster is one of the most iconic and significant buildings in the world, and home to one of the busiest parliaments, with more than a million visitors."
+      />
+      <meta
```
Comment (ildikototh) [inline src/components/SEO/SEO.js]: I am not sure that google analytics still checks this

Comment (Akallabet) [reply src/components/SEO/SEO.js]: not sure either, but it look like improve SEO performances in lighthouse

**Pull request merged**